Trip Planner/manifest.php

```diff
@@ -25,7 +25,7 @@
 $entryURL = "trips_manage.php";
 $type = "Additional";
 $category = "Learn";
-$version = "1.8.02";
+$version = "1.8.03";
```

Comment:
It looks like you've made some table changes in the CHANGEDB, also be sure to make those same changes to the table definitions in manifest.php